src/commands/test/runner.py

```diff
@@ -159,10 +158,24 @@ async def empty(
         include_paths: Optional[List[str]] = None,
     ):
         env = cls(is_test_fail_enabled, include_paths or [])
-        env.starknet = await Starknet.empty()
+        env.starknet = await ForkableStarknet.empty()
         env.test_contract = await env.starknet.deploy(contract_def=test_contract)
         return env
 
+    def fork(self):
+        assert self.starknet
+        assert self.test_contract
+
+        n_env = TestExecutionEnvironment(
```

Comment:
What does the `n` mean in this case? `new`?

Reply:
we can rename it to new